scripts/invalidate-server-cache.js

```diff
@@ -0,0 +1,14 @@
+// @ts-check
+
+require("dotenv/config")
+
+const redis = require("redis")
+
+const redisClient = redis.createClient({
+  url: process.env.OPENREDIS_URL,
+})
+
+redisClient.flushall(() => {
```

Comment:
I don't think we should flush the cache prior to releasing. There are still pods running with the old code at that point (and there will be for up to several more minutes). This causes them to slow down and increase upstream load right when we'd rather they complete their work and stop. It would be neater to just include a "digest" of the query in the cache key instead of the cache name. If you wanted to prevent cache from being shared across code versions _at all_, you'd include something that identifies the release in the cache key (like `ENV["DD_VERSION"]`). But that's aggressive... since this is a cache of Metaphysics data, there's no obvious reason a cached value from an older Force release should be invalidated.

Reply:
Cool, I'll push up a commit that removes this!

Reply:
I think we should feel it out in prod first.

Reply:
https://github.com/artsy/force/pull/14259